acceptance-tests/src/acceptance-test/java/tech/pegasys/artemis/test/acceptance/GenesisStateAcceptanceTest.java

```diff
@@ -13,13 +13,11 @@
 
 package tech.pegasys.artemis.test.acceptance;
 
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import tech.pegasys.artemis.test.acceptance.dsl.AcceptanceTestBase;
 import tech.pegasys.artemis.test.acceptance.dsl.ArtemisNode;
 import tech.pegasys.artemis.test.acceptance.dsl.BesuNode;
 
-@Disabled("Genesis generation does not yet match")
```

Comment:
So, this test passes deterministically because we're sending exactly the right number of deposits to trigger genesis?

Reply:
Yes.  The new, improved version now can handle more than the required deposits as long as they're in different blocks (ie it doesn't include all transactions from a block if only some are required to trigger genesis).